tests/libcxx/run_tests_locally.sh

```diff
@@ -16,36 +16,24 @@ FILE=$1
 while read test; do
   echo "$test"
   OUTPUT=$(2>&1 timeout 10 make one TEST=$test )
+  RETURN_VAL=$?
   echo $OUTPUT >> temp_$FILE.output
-  HAS_UNHANDLED_SYSCALL=$(2>&1 timeout 10 make one TEST=$test | grep "unhandled")
-  if [ -z "$HAS_UNHANDLED_SYSCALL" ]
+  if [[ "$RETURN_VAL" -eq "0" && -z $TERMINATED ]]
```

Comment:
Looks like a use of the `$TERMINATED` var is still lurking around.

Reply:
My bad! fixed